app/index.html

```diff
@@ -1,7 +1,14 @@
+<!DOCTYPE html>
 <html>
-  <link rel="stylesheet" type="text/css" href="dist.css">
-  <title>Task Editor</title>
+  <head>
+    <base target="_blank">
+    <meta charset="UTF-8">
+    <title>Simple Task Editor</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" media="all" href="dist.css">
+  </head>
   <body>
+    <div class="sn-component" id="simple-task-editor"></div>
```

Comment:
@hyphone what's the basis for this line?

Reply:
@mobitar none really - I just played around and thought maybe this element with the sn-component class would be necessary to embed the component properly (and perhaps scaling would magically work). Should I remove it and make a new proper pull request with just the viewport?

Reply:
Yes you can remove it in the same PR..